docs/webapi/checkOption.mustache

```diff
@@ -9,4 +9,4 @@ I.checkOption('I Agree to Terms and Conditions');
 I.checkOption('agree', '//form');
 ```
 @param {string|object} field checkbox located by label | name | CSS | XPath | strict locator.
-@param {string} context (optional, `null` by default) element located by CSS | XPath | strict locator.
\ No newline at end of file
+@param {string} [context=null] (optional, `null` by default) element located by CSS | XPath | strict locator.
```

Comment:
yeah, technically you are correct here. However, I had issues generating documentation when defining optional types like this. I spend a day trying to fix documentation rebuild but I found that better solution for now not to use JSDoc styles and write manually `optional` word

Reply:
Is the correct implementation of my code fine, or would you like to something else?